examples/withClientWithTempManager.js

```diff
@@ -21,7 +21,9 @@ client.on('ready', () => {
     childAutoDeleteIfOwnerLeaves: false,
     childAutoDeleteIfParentGetsUnregistered: true,
     childVoiceFormat: (str, count) => `Example #${count} | ${str}`,
-    childVoiceFormatRegex: /^Example #\d+ \|/
+    childVoiceFormatRegex: /^Example #\d+ \|/,
+    childMaxUsers: 0,
```

Comment:
```suggestion
    childMaxUsers: 3,
```

Reply:
Why? Is 0 not unlimited?

Reply:
It is, but it also feels wrong as a value. I'd rather set an explicit value and people simply remove the line than set it to 0 and people get confused.

Reply:
Hmm, yeah

Reply:
Fixed! See 054dc56